compatibility/versions/UpdateVersions.hs

```diff
@@ -35,7 +35,7 @@ instance Semigroup Versions where
     Versions a <> Versions b = Versions (a <> b)
 
 minimumVersion :: Version
-minimumVersion = SemVer.incrementMajor SemVer.initial
+minimumVersion = SemVer.version 2 3 0 [] []
```

Comment:
That seems unrelated to the issue of including 2.5.0 - would you mind commenting on the change here? Per [the documentation](https://hackage.haskell.org/package/semver-0.4.0.1/docs/Data-SemVer.html) it looks like this previously evaluated to `1.0.0`, so switching to `2.3.0` seems like a big change.